pymc/logprob/transforms.py

```diff
@@ -657,7 +657,8 @@ def find_measurable_transforms(fgraph: FunctionGraph, node: Node) -> Optional[Li
 
     # Do not apply rewrite to discrete variables
     if measurable_input.type.dtype.startswith("int"):
-        return None
+        if str(node.op) != "Mul" and str(node.op) != "Add":
+            return None
```

Comment:
What does this do?

Reply:
It basically allows discrete rvs which involves multiplication and addition instead of rejecting all discrete rvs.